test/UnitTests/StagingTest.cpp

```diff
@@ -21,16 +21,16 @@
 #include "RepositoryFixtures.h"
 #include "StagingOperations.h"
 
-inline std::unique_ptr<char[]> ReadResourcePatchToBuffer(LPCTSTR resourceFile)
+inline char* ReadResourcePatchToBuffer(LPCTSTR resourceFile)
 {
 	CString resourceDir;
 	GetResourcesDir(resourceDir);
 
 	CFile file;
 	file.Open(resourceDir + resourceFile, CFile::modeRead | CFile::typeBinary);
 	auto filelen = file.GetLength();
-	auto buf = std::make_unique<char[]>(static_cast<size_t>(filelen) + 1);
-	file.Read(buf.get(), static_cast<UINT>(filelen));
+	auto buf = new char[static_cast<size_t>(filelen) + 1]();

```

Comment:
Here:
* Is this copy of the string realyl necessary?
* Please keep using the `unique_ptr` here.